### ISSUE TITLE
[A11y][Contrast]"Organizations"  and "upload" Menu item are not visible in high contrast black

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8187]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1184516

## Browser Details:
Firefox Version: 79.0 (64 bit)

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450)

## Repro Steps:

    Turn on High contrast Black mode
    Hit the URL and login with appropriate credentials
    Tab till "Open Profile drop down" and press enter
    Tab till "Manage Organizations" and Press enter
    Check whether all the text in the page is clearly visible and"Organizations"  and "upload" Menu item are  visible in high contrast black or not

## Actual Results:
"Organizations"  and "upload" Menu item are not visible in high contrast black
## Expected Results:
"Organizations"  and "upload" Menu item should be visible in high contrast black mode
## User Impact:
high contrast users may find difficulty to see the "Organizations" link and 'Upload" menu item

## Notes
There should be high contrast settings in the Styles. Identifying which are used here, or adding new ones for the given scenario will be required for fix.

**Comment by loic-sharma** [comment]
This appears to be a Firefox bug (see my commenet in the AzDO work item linked above). I marked this as "Won't fix" but will wait for accessibility folks to reply.
